### DISCUSSION TITLE
Days Delay

### DISCUSSION THREAD

**Discussion by rschmetzer** [moneymanagerex/moneymanagerex#6989] [Q&A]
How can I create a future annual transaction in Checking without generating an immediate xx Days Delay entry under Upcoming Transactions on the dashboard?  It always seems to take into account the date I'm creating the transaction when that has nothing to do with the transaction date.

**Comment by rmelillo76** [comment]
Hi @rschmetzer There are 2 dates in the scheduled transaction - 'Date Due' and  'Date Paid'. 
The  'Date Paid' field is the one that drives the item to be displayed when that date is within 14 calendar days from today, so you will want to have that be the future expected date.

If you are still having issues, could you paste a screenshot of the scheduled transaction here as well as the Upcoming Transaction table on the dashboard?

**Comment by rschmetzer** [comment]
Okay, Changing the Date Paid field fixed my issue.  Just don't understand
why Date Due isn't the driving item....

**Reply by rmelillo76** [reply]
> Okay, Changing the Date Paid field fixed my issue. Just don't understand why Date Due isn't the driving item....

Good to know. I'm not sure either, don't know if it's a quirk or there is some logical reasoning behind it.
